lib/farscape/version.rb

```diff
@@ -3,8 +3,8 @@
   module Farscape
     module VERSION
```

Comment:
can you get rid of this and just do:
```ruby
VERSION = '1.1.0'.freeze
```
Then you will need to change the gemspec to use Farscape::Version instead of Farscape::VERSION::STRING